src/framework/preferences/css/stylus/Enactors.styl

```diff
@@ -26,14 +26,14 @@
 }
 
 // UI Enhancer "Enlarge text" input/button options
-.fl-text-larger input,
-.fl-text-larger button,
-.fl-text-larger .fl-input-enhance {
+.fl-input-enhanced input,
```

Comment:
We talked more about this in the [IRC Channel](https://botbot.me/freenode/fluid-work/2017-04-12/?msg=83924100&page=2). The conclusion was to just add styles for `<textarea>` and `<select>`.